core/src/main/scala/zio/schema/Schema.scala

```diff
@@ -16,24 +19,37 @@ sealed trait Schema[A] { self =>
 }
 
 object Schema {
-  sealed case class Record(structure: Map[String, Schema[_]])      extends Schema[Map[String, _]]
-  sealed case class Sequence[A](element: Schema[A])                extends Schema[Chunk[A]]
+
+  sealed case class Record(structure: Map[String, Schema[_]]) extends Schema[Map[String, _]]
+
+  sealed case class Sequence[A](element: Schema[A]) extends Schema[Chunk[A]]
+
   sealed case class Enumeration(structure: Map[String, Schema[_]]) extends Schema[Map[String, _]]
+
   sealed case class Transform[A, B](codec: Schema[A], f: A => Either[String, B], g: B => Either[String, A])
       extends Schema[B]
-  sealed case class Primitive[A](standardType: StandardType[A])          extends Schema[A]
-  sealed case class Optional[A](codec: Schema[A])                        extends Schema[Option[A]]
-  final case class Fail[A](message: String)                              extends Schema[A]
-  sealed case class Tuple[A, B](left: Schema[A], right: Schema[B])       extends Schema[(A, B)]
+
+  sealed case class Primitive[A](standardType: StandardType[A]) extends Schema[A]
+
+  sealed case class Optional[A](codec: Schema[A]) extends Schema[Option[A]]
+
+  final case class Fail[A](message: String) extends Schema[A]
+
+  sealed case class Tuple[A, B](left: Schema[A], right: Schema[B]) extends Schema[(A, B)]
+
   final case class EitherSchema[A, B](left: Schema[A], right: Schema[B]) extends Schema[Either[A, B]]
 
+  final case class ListSchema[A](codec: Schema[A]) extends Schema[List[A]]
```

Comment:
@ashprakasan Sorry I missed this, since it was also in the other PR. We should delete ListSchema and implement the schema for List in terms of the existing Schema.Sequence.

Reply:
Just noticed that this [method](https://github.com/zio/zio-schema/blob/main/core/src/main/scala/zio/schema/Schema.scala#L814-L815) already exists.

However in the evaluation of `Remote` (for zio/zio-flow), I need to get `Schema[A]` from `Schema[List[A]]`. Not sure how to do that without introducing ListSchema